github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -2066,3 +2066,63 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - alwways_run: false
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    cluster: kubevirt-prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 4h0m0s
+    labels:
+      ci.kubevirt.io/prometheus: ""
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-podman-in-container-enabled: "true"
+      preset-podman-shared-images: "true"
+    name: pull-kubevirt-e2e-k8s-1.29-sig-performance-kwok
+    optional: true
+    reporter_config:
+      slack:
+        job_states_to_report: []
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - |
+          # Build the perfscale-audit binary
+          make bazel-build
+          # Create k8s cluster
+          make cluster-up
+          # Push and deploy kubevirt
+          make cluster-sync
+          FUNC_TEST_ARGS="--no-color --seed=42" make kwok-perftest
+        env:
+        - name: KUBEVIRT_PROVIDER
+          value: k8s-1.29
+        - name: KUBEVIRT_STORAGE
+          value: hpp
```

Comment:
Do we actually need storage?

Reply:
@xpivarc I believe logs and artifacts are stored in KUBEVIRT_STORAGE. Is this storage configuration specifically for storage within the cluster? The logs appear to be on the pod executing the test, which is external to the cluster, are these logs not being stored?

Reply:
No, the `KUBEVIRT_STORAGE` is for in cluster storage. Logs as objects,logs etc are collected and pushed to bucket. @rthallisey @alaypatel07 do we need storage for anything?

Reply:
Let's drop this, we merge and try in kubevirt. We can come back if something fails...

Reply:
I am not sure, @Sreeja1725 we can start with removing the storage and merge the PR, we can change it in future if needed.

Reply:
Removed the KUBEVIRT_STORAGE for now